windows-container-samples/ASP-NET-Blog-Application/db/dockerfile

```diff
@@ -0,0 +1,22 @@
+
+# SQL Express Base Image 
+# NOTE: You may need to prepend 'microsoft/' to base image name--use 'docker images' to see what your image is called.
+FROM microsoft/mssql-server-2014-express-windows
+
+# SQL environmental variables (replace "Password123" with your sa password)
+ENV sa_password "Password123"
```

Comment:
This can also be done from the compose file: https://github.com/aspnet/MusicStore/pull/713/files#diff-7ba4a03630a062fa2698551ad3cf2109R4